packages/runtime/datastore/api-report/datastore.legacy.alpha.api.md

```diff
@@ -70,8 +70,8 @@ export class FluidDataStoreRuntime extends TypedEventEmitter<IFluidDataStoreRunt
     get objectsRoutingContext(): this;
     // (undocumented)
     readonly options: Record<string | number, any>;
-    // (undocumented)
     process(message: ISequencedDocumentMessage, local: boolean, localOpMetadata: unknown): void;
+    processMessages(message: ISequencedRuntimeMessageCore, messageContents: IRuntimeMessageContents[], local: boolean): void;
```

Comment:
FYI - I've left some comments on the type names / packaging of the args in DDS PR

Reply:
I updated the types and names here. Please take a look.